docs/conf.py

```diff
@@ -234,3 +245,217 @@ def setup(app):
+            \renewcommand{\footrulewidth}{0pt}
+            \fancyfoot[C]{{\small\thepage}}
+        }
+        % normal = subclass of plain = applies to rest of pages
+        \fancypagestyle{normal}{
+            \fancyhead{}
+            \fancyhead[L]{\small\rightmark}
+            \fancyhead[R]{{\small\thepage}}
+            \renewcommand{\headrulewidth}{0.4pt}
+            \renewcommand{\footrulewidth}{0pt}
+            \fancyfoot[C]{{}}  % undo the \fancyfoot[C]{{\small\thepage}} in the base class
+            %   \fancyfoot[R]{\ifthenelse{\isodd{\value{page}}}{{\tiny Learning Equality} }{\href{https://learningequality.org}{\tiny learningequality.org}}}
+            %%% Alternating Footer for two side
+            %%% \fancyfoot[RO, RE]{\scriptsize Learning Equality (info@learningequality.org)}
+        }
+        %% callback that changes document header text
+        %%                                               = section number counter (\thesection)
+        %%                                                          + the section title
+        \renewcommand{\sectionmark}[1]{\markboth{}{\emph{\thesection~#1}}}
+        %%%% optional: uncomment next line to change header on every \section and not on \subsection
+        %%%% \renewcommand{\subsectionmark}[1]{}
+
+
+        %%reduce spacing for itemize ???
+        \usepackage{enumitem}
+        \setlist{nosep}
+
+        %%%%%%%%%%% Quote Styles at the top of chapter ???
+        \usepackage{epigraph}
+        \setlength{\epigraphwidth}{0.8\columnwidth}
+        \newcommand{\chapterquote}[2]{\epigraphhead[60]{\epigraph{\textit{#1}}{\textbf {\textit{--#2}}}}}
+        %%%%%%%%%%% Quote for all places except Chapter
+        \newcommand{\sectionquote}[2]{{\quote{\textit{``#1''}}{\textbf {\textit{--#2}}}}}
+    ''',
+
+    'maketitle': r'''
+
+        %%% SET PDF INFO %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+        % Note: this has to be after preamble where \title and \author are defined
+        \makeatletter
+        \hypersetup{
+            pdftitle=\@title,
+            pdfauthor=\@author,
+            pdfkeywords={Kolibri, Kolibri Studio, offline learning, open educational resources},
+            pdfsubject={Kolibri},
+            pdfstartview=Fit,   % default value
+            pdfstartpage=1,     % default value
+            % other available options:
+            % pdfstartview={FitH},
+            % pdfpagelayout=SinglePage,
+            % pdfpagelabels=true,
+            % bookmarksopen=true,
+            % bookmarksopenlevel=0
+        }
+        \makeatother
+
+        %%%  COVER PAGE  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+        \pagenumbering{Roman} %%% to avoid page 1 conflict with actual page 1
+
+        \begin{titlepage}
+            \centering
+
+            \vspace*{80mm} %%% * is used to give space from top
+            \begin{figure}[!h]
+            \centering
+            \includegraphics[scale=0.75]{logo.png}
```

Comment:
This would make a better cover-page logo since it shows contexts of use:
https://github.com/learningequality/kolibri-docs/blob/develop/docs/img/kolibri-use-cases.png

Reply:
@khangmach do we have an SVG copy of this?

Reply:
No, there's only a PNG format at the moment. Do we need an svg?

Reply:
if available, it would come out nicer when printed

Reply:
Doesn't need to be this either. I think we're just choosing an illustration to put on the front

Reply:
That's precisely what I wanted to use on the cover page @khangmach (see the sample doc above), unless you have a better suggestion...?

In any case, SVG version of whatever we decide to use would be great to have!

Reply:
yeah i'll spin up an SVG version of that illustration

Reply:
should I forward it to you on slack or

Reply:
cool, maybe post in the `#documentation` room